src/components/CreateBeer.js

```diff
@@ -0,0 +1,143 @@
+import { useState } from "react";
+
+// Iteration 4
+function CreateBeer(props) {
+  const [name, setName] = useState("");
+  const [tagline, setTagline] = useState("");
+  const [description, setDescription] = useState("");
+  const [firstBrewed, setFirstBrewed] = useState("");
+  const [brewersTips, setBrewersTips] = useState("");
+  const [attenuationLevel, setAttenuationLevel] = useState(0);
+  const [contributedBy, setContributedBy] = useState("");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const newBeer = {
+      name: name,
+      tagline: tagline,
+      description: description,
+      first_brewed: firstBrewed,
+      brewers_tips: brewersTips,
+      attenuation_level: attenuationLevel,
+      contributed_by: contributedBy,
+    };
+    // update list of Beers in local
+    // props.setBeers((prevBeers) => {
+    //   return [newBeer, ...prevBeers];
+    // });
+
+    // update list of Beers in API!!!!!!!!
+    props.createBeer(newBeer);
+
+    // clear form
+    // setName("");
+    // setTagline("");
+    // setDescription("");
+    // setFirstBrewed("");
+    // setBrewersTips("");
+    // setAttenuationLevel(0);
+    // setContributedBy("");
```

Comment:
You are correct here clearing the form :) and after could be a good idea to redirect the user and giving the user a feedback message if the beer was created correctly or giving the user the feedback + the choice if they want to create another beer or being redirect :)